descriptor.md

```diff
@@ -28,13 +28,14 @@ The following describe the primary set of properties that make up a _Descriptor_
   This property exists so that a client will have an expected size for the content before validating.
   If the length of the retrieved content does not match the specified length, the content SHOULD NOT be trusted.
 
-### Reserved
+- **`urls`** *array*
 
-The following are field keys that MUST NOT be used in descriptors specified in other OCI specifications:
+  This OPTIONAL property specifies a list of URLs from which this object can be downloaded.
```

Comment:
“object” → “blob” (see [this][1] and later).

[1]: https://github.com/opencontainers/image-spec/pull/94#discussion_r64955841